Sources/SwiftUI/KFAnimatedImage.swift

```diff
@@ -75,6 +76,35 @@ public struct KFAnimatedImageViewRepresenter: UIViewRepresentable, KFImageHoldin
         uiView.image = image
     }
 }
+#else
+@available(macOS 11.0, *)
+public struct KFAnimatedImageViewRepresenter: NSViewRepresentable, KFImageHoldingView {
```

Comment:
I see quite a lot of duplicated code.

Maybe we can combine these two `KFAnimatedImageViewRepresenter`s into one? Something like:

```swift
#if !os(macOS)
typealias KFCrossPlatformViewRepresentable = UIViewRepresentable
#else
typealias KFCrossPlatformViewRepresentable = NSViewRepresentable
#endif
```

What your opinion?

Reply:
Good idea. Updated